src/main/java/org/openrewrite/java/testing/assertj/SimplifyAssertExpectations.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.assertj;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import org.openrewrite.java.template.RecipeDescriptor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RecipeDescriptor(
+        name = "Simplify the expectations of assertions",
+        description = "Simplifies expectations on various assertions."
+)
+public class SimplifyAssertExpectations {
+
+    @RecipeDescriptor(
+            name = "Simplify `assertThat(int).isEqualTo(0)`",
+            description = "Simplify `assertThat(int).isEqualTo(0)` to `assertThat(int).isZero()`."
+    )
+    public class SimplifyToIsZero {
+
+        @BeforeTemplate
+        void before(Integer i) {
```

Comment:
I think you want to use primitive types in the method signatures.

Reply:
I tried it for both "Integer" recipes, but still get "Recipe was expected to make a change but made no changes." for all tests.